src/main/java/org/cirdles/topsoil/app/utils/SVGSaver.java

```diff
@@ -37,32 +43,74 @@
     public static final String SVG_DOCTYPE_PUBLIC = "-//W3C//DTD SVG 1.1//EN";
     private static final String SVG_DOCTYPE_SYSTEM = "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd";
 
+    /**
+     * Saves the vector image in Topsoil with the .svg file extension
+     *
+     * @param svgDocument   Document to be saved
+     */
     public void save(Document svgDocument) {
+
+        // If there is no file specified, then there is nothing to do.
+        // Otherwise, write the file with the SVG format.
+        generateSaveUI()
+                .ifPresent(stream -> {
+                    writeSVGToOutputStream(svgDocument, stream);
+                });
+    }
+
+    /**
+     * Create a save dialog window and prompt user for filepath
+     *
+     * @return Optional<File>   the filepath
+     */
+    private Optional<OutputStream> generateSaveUI() {
         // get the path for the new SVG file
         FileChooser fileChooser = new FileChooser();
         fileChooser.setTitle("Export to SVG");
         fileChooser.setInitialDirectory(new File(System.getProperty("user.home")));
-        fileChooser.getExtensionFilters().add(new FileChooser.ExtensionFilter("SVG Image", "*.svg"));
-        File svgFile = fileChooser.showSaveDialog(null);
+        fileChooser.getExtensionFilters().setAll(
+                new FileChooser.ExtensionFilter("All Files", "*"),
+                new FileChooser.ExtensionFilter("SVG Image", "*.svg")
+        );
 
-        // can't convert if the file isn't specified
-        if (svgFile == null) {
-            return;
+        Optional<File> file = Optional.ofNullable(fileChooser.showSaveDialog(null));
+        if (file.isPresent()) {
+            try {
+                Optional<OutputStream> stream = Optional.ofNullable(new FileOutputStream(file.get()));
+                //Necessary for clean up
+                try {
+                    stream.get().close();
```

Comment:
This looks like a bug. Are you sure that contents are actually being written to file?